cppwamp/include/cppwamp/internal/udsconnector.ipp

```diff
@@ -9,7 +9,6 @@
 #include "asioconnector.hpp"
 #include "client.hpp"
 #include "config.hpp"
-#include "udsopener.hpp"
```

Comment:
Why did you remove? `internal::UdsOpener` is used everywhere in this source file.